### DISCUSSION TITLE
Kerberos almost working, but missing something....

### DISCUSSION THREAD

**Discussion by craigboyd0** [dbeaver/dbeaver#19710] [Q&A] [answered]
#### System information: 
- Operating system Windows 10
- DBeaver version 21.1.5
- Additional extensions

#### Connection specification:
- Database name and version Oracle 19c
- Driver name  ojdbc11-21.5.0.0.jar
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No
- Using Kerberos Authentication

#### Describe the problem you're observing:
Getting invalid username / password
ORA-01017: invalid username/password; logon denied

#### Steps to reproduce, if exist:
Main tab, Custom, JDBC URL: {contains a working JDBC URL string otherwise I would be getting a different error}
Driver Properties: 
oracle.net.kerberos5_cc_name {c:\users\username\kerberos_ticket_file (created using kinit)
oracle.net.kerberos5_mutual_authentication true

Due to company firewall issues I had to download a copy of the Oracle driver and point to it, find and choose the class (oracle.jdbc.OracleDriver).
No other settings have been made.

Any idea what is going on? I feel like I am missing something simple since I am getting that ORA error.
Thanks.

**Comment by Matvey16** [comment, marked as answer]
Unfortunately we can't help you with kerberos authentication through driver properties because it can vary depending on your work environment. I can suggest to try using DBeaver Enterprise Edition which has dedicated kerberos authentication settings and comes with a free 2 weeks trial period. If you want to continue using Community version I suggest searching JDBC documentation which can contain needed information
